### DISCUSSION TITLE
Could the month arrow active style be turned off after click?

### DISCUSSION THREAD

**Discussion by VilleKylmamaa** [gpbl/react-day-picker#1528] [Support] [answered]
When you click the arrow to change the month, the button will remain in focus and the focus style will remain. It's good when you are using the keyboard to navigate but I find it odd when using the mouse.

Could the focus or just the style be dropped on click?

**Comment by gpbl** [comment]
@VilleKylmamaa thanks for reporting this – notably it doesn't happen with safari. Looks like switching from `:focus` to `:focus-visible` here: https://github.com/gpbl/react-day-picker/blob/master/packages/react-day-picker/src/style.css#L61 fixes it. 

I will investigate this soon - meanwhile, you may want to override that selector as:

```diff
- .rdp-button:focus:not([disabled]),
+ .rdp-button:focus-visible:not([disabled]),
.rdp-button:active:not([disabled]) {
  color: inherit;
  border: var(--rdp-outline);
  background-color: var(--rdp-background-color);
}
```

**Comment by gpbl** [comment, marked as answer]
This is fixed via https://github.com/gpbl/react-day-picker/pull/1531 in v8.1.4 – please report back if it does not, thanks!
